test/travis/firebase_init.cc.snip

```diff
@@ -0,0 +1,28 @@
+// Set these to run example.
```

Comment:
We will eliminate all of the .snips in the medium term. SerialHost is moving to it's own project at some point and the other examples should be ported over to use the FirebaseArduino class since that is now our go-to for Arduino projects.